requirements.txt

```diff
@@ -1,4 +1,74 @@
-cython>=0.29.20
-numpy>=1.16.6
-scipy>=1.0
-packaging
+alabaster==0.7.12
+asttokens==2.0.5
+attrs==21.4.0
+Babel==2.9.1
+backcall==0.2.0
+black==21.12b0
+build==0.7.0
+certifi==2020.6.20
+charset-normalizer==2.0.10
+click==8.0.3
+cycler==0.11.0
+Cython==0.29.26
+decorator==5.1.1
+docutils==0.16
+executing==0.8.2
+fonttools==4.28.5
+idna==3.3
+imagesize==1.3.0
+iniconfig==1.1.1
+ipython==8.0.0
+jedi==0.18.1
+Jinja2==3.0.3
+kiwisolver==1.3.2
+latexcodec==2.0.1
+MarkupSafe==2.0.1
+matplotlib==3.4.3
+matplotlib-inline==0.1.3
+mypy-extensions==0.4.3
+numpy==1.22.0
+numpydoc==1.1.0
+packaging==21.3
+parso==0.8.3
+pathspec==0.9.0
+pep517==0.12.0
+pexpect==4.8.0
+pickleshare==0.7.5
+Pillow==9.0.0
+platformdirs==2.4.1
+pluggy==1.0.0
+prompt-toolkit==3.0.24
+ptyprocess==0.7.0
+pure-eval==0.2.1
+py==1.11.0
+pybtex==0.24.0
+pybtex-docutils==1.0.1
+Pygments==2.11.2
+pyparsing==3.0.6
+pytest==6.2.5
+pytest-rerunfailures==10.2
+python-dateutil==2.8.2
+pytz==2021.3
+PyYAML==6.0
+qutip @ file:///home/anna_user2/Dropbox/projects/sphinx-warnings/qutip/dist/qutip-4.7.0.dev0%2Bnogit-cp310-cp310-linux_x86_64.whl
```

Comment:
Hey @anna-naden great work with the review of the documentation! I have been following along the different PR you opened but did not have time to review them properly. I did catch in this one something off. Specifically this line should not be included as it would fail in any other machine.

This requirements file is seldom use but it still only refers to the runtime requirements (see PR #1715 where it was last modified). There is another requirements file in `doc/` that does have the requirements for building the documentation (which maybe was what you were trying to actually modify?).

 In any case, pinning an exact version of a package is not a good idea as it is too restrictive. For example, different packages may have different requirements for numpys version. Forcing an specific version here means that it will conflict with other packages. Hence, I suggest removing the changes in this file.